src/globals.cpp

```diff
@@ -348,6 +348,10 @@ CTweak<bool> enableCanonicalTxOrder("consensus.enableCanonicalTxOrder",
 
 CTweak<unsigned int> numMsgHandlerThreads("net.msgHandlerThreads", "Max message handler threads", 0);
 CTweak<unsigned int> numTxAdmissionThreads("net.txAdmissionThreads", "Max transaction mempool admission threads", 0);
+CTweak<unsigned int> unconfPushAction("net.unconfChainResendAction",
+    "Action to take when this node thinks that a peer will now accept a previously unacceptable unconfirmed transaction"
+    "0: do not resend, 1: send an INV, 2: send the TX",
+    0);
```

Comment:
I wonder whether the INV option would even work because the tx might be in recentrejects and so we wouldn't request it again?

Reply:
Yes, that's a problem -- its basically a bug in full node implementations to continue to reject transactions that would now be valid.  But it is not affecting this PR since this code does not forward the transaction if it won't be accepted. So it won't be in the recent rejects (unless some other poorly-behaving node forwards it).